src/components/autocomplete/js/autocompleteController.js

```diff
@@ -527,6 +535,38 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Determines if the escape keydown should be processed
+   * @returns {boolean}
+   */
+  function shouldProcessEscape() {
+    if (shouldBlurOnEscape()) return true;                            // Needs blur
+    else if (!ctrl.hidden) return true;                               // Hide dropdown
+    else if (ctrl.loading) return true;                               // Cancel loading
+    else if (shouldClearOnEscape() && $scope.searchText) return true; // Clear Text
+    else return false;
+  }
+
+  /**
+   * Determines if the escape key should blur focus
+   * @returns {boolean}
+   */
+  function shouldBlurOnEscape() {
+    if (!$scope.escapeOptions) return true;                     // default true
+    if ($scope.escapeOptions.indexOf('blur') >= 0) return true; // blur is requested
```

Comment:
I saw using another piece of code for reference but I can easily add `toLowerCase`